docker/ol-web1-start.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
Can we avoid a new file here? Maybe some env variables for `USE_HAPROXY` and `USE_NGINX`?